### DISCUSSION TITLE
OWL Gen2.0 Workshop Tomorrow!

### DISCUSSION THREAD

**Discussion by geezacoleman** [geezacoleman/OpenWeedLocator#135] [Announcements]
Tomorrow at 3pm AWST (UTC +8), I'll be running an online workshop for the latest version of the OWL. It will include a brief demonstration on a treadmill and introduction of the latest controller.

You can register [here on Humanitix](https://events.humanitix.com/openweedlocator-gen2-0-workshop).

Hope to see as many of you there as possible. As with the last workshop, this will also be recorded and uploaded to the [Open AgTech Youtube Channel.](https://www.youtube.com/channel/UCQxrZOfuLxlNM1i-gAueoLw)

Cheers,
Guy

**Comment by Timmy211** [comment]
Hi Guy, 

I saw you present this at EvokeAg in February and have been following the OWL ever since.  

I watched the recording on YouTube.  I had been waiting on your recent advancements with the RPi 5 before buying the parts to start making a set up.  I am keen to give this a go across a larger spray boom for broadacre spraying.  I will start with one Rpi 5 and camera to get it working and then look at going bigger.  Some parts are here, just need to get a few more things.

Interested in the ideas you are developing for in-cab control and gps integration as well.

This is awesome stuff.

Cheers,
Tim

**Reply by geezacoleman** [reply]
Hi Tim,

This is exciting to hear! Thanks for following along. How wide do you think you'll go? We're planning on building a medium sized 12m three-point linkage system in Denmark later this year, so I'll share some of the developments/learnings from that too.

Current in-cab control is just GPIO-pin based and limited to probably 4m, but will move to a Raspberry Pi/screen + switches and MQTT to manage each connected OWL over an ethernet network. Were considering CANBUS but I think ethernet for the moment because of its versatility. Can tap into the video feeds too.

Let me know if you need any aluminium enclosures too (email is guy[at]openag.tech) or run into any issues. Looking forward to seeing how you go! Always open to suggestions/fixes/improvements, so just flick me an email or raise an issue here and I'll do my best to respond.

Cheers,
Guy

**Comment by geezacoleman** [comment]
The recording for the workshop is now available on the [Open AgTech YouTube Channel](https://www.youtube.com/watch?v=2CWd1HVJdTs).

Thanks to everyone that took part!
